tools/publish.sh

```diff
@@ -32,14 +32,15 @@ crates=(
     bevy_internal
     bevy_dylib
 )
+set -eou pipefail
```

Comment:
This means that in case of failure to publish crates, it will stop on the first failure instead of ignoring and continuing for other crates. I don't know the intended behaviour as it's a tool used only by @cart though.

Reply:
yeah i like the current behavior, lets keep that for now.